juniper_official/Kernel/check-too-many-dead-ifstates-iagent.rule

```diff
@@ -44,12 +44,10 @@ healthbot {
                 }
                 type integer;
             }
-            field ratio {
+            field alive-half-cnt {
                 formula {
-                    user-defined-function {
-                        function-name get-dead-alive-ratio;
-                        argument alive_ifstates_cnt "$alive-cnt";
-                        argument dead_ifstates_cnt "$dead-cnt";
+                    eval {
```

Comment:
Python function to be removed from rule as well python file. But ensure that no other rule is using python file.
Same applies to all other rules and python files

Reply:
Removed the functions from the rules and deleted the pytrhon files that are not being used.